2018-02-13T12:51:00.554Z - info: Test execution started
2018-02-13T12:51:00.554Z - info: Got following options to run{"testName":"Clean Install","UDID":"cd21ccc5","port":4004,"systemPort":8204,"bpPort":6004,"desCaps":"{\n\"testName\":\"Clean Install\",\n\"groupTest\":false,\n\"desiredCapabilities\": {\n        \"platformName\": \"Android\",\n        \"deviceName\": \"Android\",\n        \"appPackage\":\"com.gorro.nothing\",\n        \"appActivity\":\"com.gorro.nothing.NothingActivity\",\n        \"fullReset\":false,\n        \"newCommandTimeout\":120000,\n        \"skipUnlock\":true\n      }\n\n}\n"}
2018-02-13T12:51:00.562Z - info: Trying to connect to openSTF with serial number cd21ccc5
2018-02-13T12:51:00.611Z - info: executing command adb connect  localhost:7409
2018-02-13T12:51:00.619Z - info: /home/trixo/Downloads/appium-1.8.0-beta3/build/lib/main.js -p 4004 -bp6004  --log-timestamp --log /home/trixo/Code/javaRewrite/log4004.txt
2018-02-13T12:51:00.619Z - info: Trying to create appium Server on port: 4004
2018-02-13T12:51:02.762Z - info: Appium server created on port: 4004
2018-02-13T12:51:23.621Z - info: Running clean Install test with following paramters, appPackage: com.pointvoucher.playlondonpv activityName: com.unity3d.player.UnityPlayerActivity apkFile: /home/trixo/Code/javaRewrite/Tests/CleanInstall/apk/pl.apk
2018-02-13T12:51:25.384Z - info: App exists in device => trying to remove
2018-02-13T12:51:31.690Z - info: App removed
2018-02-13T12:51:31.690Z - info: Installing app to device /home/trixo/Code/javaRewrite/Tests/CleanInstall/apk/pl.apk
2018-02-13T12:54:33.901Z - info: starting package : com.pointvoucher.playlondonpv with activity name: com.unity3d.player.UnityPlayerActivity
2018-02-13T12:54:38.255Z - info: Running fnPermission with number of repeats: 5 and value: true
2018-02-13T12:54:39.590Z - info: Found permission button going to click #0
2018-02-13T12:54:39.590Z - info: Running click event : Click on permission button with 5repets
2018-02-13T12:54:39.591Z - info: Looking for image on screen:Click on permission button with 5 repeats 
2018-02-13T12:54:39.591Z - info: Running delay for : 0
2018-02-13T12:54:39.593Z - info: Running delay for : 0
2018-02-13T12:54:40.822Z - info: Found image on screen: Click on permission button
2018-02-13T12:54:41.193Z - info: clicked on : Click on permission button with following coordinates x=438 y=556.5
2018-02-13T12:54:41.194Z - info: Looking for image on screen:Loading screen with 20 repeats 
2018-02-13T12:54:41.194Z - info: Running delay for : 6000
2018-02-13T12:54:47.196Z - info: Running delay for : 2000
2018-02-13T12:54:50.539Z - info: Running delay for : 2000
2018-02-13T12:54:53.474Z - info: Running delay for : 2000
2018-02-13T12:54:56.543Z - info: Running delay for : 2000
2018-02-13T12:54:59.342Z - info: Running delay for : 2000
2018-02-13T12:55:02.509Z - info: Running delay for : 2000
2018-02-13T12:55:05.431Z - info: Running delay for : 2000
2018-02-13T12:55:08.726Z - info: Running delay for : 2000
2018-02-13T12:55:12.285Z - info: Running delay for : 2000
2018-02-13T12:55:15.940Z - info: Running delay for : 2000
2018-02-13T12:55:19.109Z - info: Running delay for : 2000
2018-02-13T12:55:22.921Z - info: Found image on screen: Loading screen
2018-02-13T12:55:22.921Z - info: Test:Clean Install[FINISHED]
2018-02-13T12:55:22.943Z - info: Disconnected localhost:7409=> open STF 
2018-02-13T12:55:22.948Z - info: Appium err 2018-02-13 12:55:22:948 - [Logcat] Logcat terminated with code 0, signal null

2018-02-13T12:55:22.949Z - info: Appium err 2018-02-13 12:55:22:949 - [UiAutomator] UiAutomator exited unexpectedly with code 0, signal null

2018-02-13T12:55:22.981Z - info: Appium err 2018-02-13 12:55:22:981 - [Appium] Had trouble ending session 627ea621-f153-405b-a382-802cbb4a7a79: Error executing adbExec. Original error: 'Command '/home/trixo/Android/Sdk/platform-tools/adb -P 5037 -s localhost\:7409 shell am force-stop com.gorro.nothing' exited with code 1'; Stderr: 'error: device 'localhost:7409' not found'; Code: '1'

2018-02-13T12:55:23.001Z - info: Appium err 2018-02-13 12:55:23:000 - [MJSONWP] Encountered internal error running command: Error: Error executing adbExec. Original error: 'Command '/home/trixo/Android/Sdk/platform-tools/adb -P 5037 -s localhost\:7409 shell am force-stop com.gorro.nothing' exited with code 1'; Stderr: 'error: device 'localhost:7409' not found'; Code: '1'
    at ADB.execFunc$ (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/appium-adb/lib/tools/system-calls.js:317:13)
    at tryCatch (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:136:37)
    at <anonymous>

2018-02-13T12:55:23.011Z - info: Appium err Unhandled rejection Error: Error executing adbExec. Original error: 'Command '/home/trixo/Android/Sdk/platform-tools/adb -P 5037 -s localhost\:7409 shell am force-stop com.gorro.nothing' exited with code 1'; Stderr: 'error: device 'localhost:7409' not found'; Code: '1'
    at ADB.execFunc$ (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/appium-adb/lib/tools/system-calls.js:317:13)
    at tryCatch (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (/home/trixo/Downloads/appium-1.8.0-beta3/node_modules/babel-runtime/regenerator/runtime.js:136:37)
    at <anonymous>

2018-02-13T13:02:14.793Z - info: Appium on close  null
2018-02-13T13:02:14.803Z - info: You are not owner localhost:7409 => open STF 
2018-02-13T13:02:14.806Z - info: You are not owner localhost:7409 => open STF 
